README.md

```diff
@@ -32,7 +36,7 @@ TrueColors and the included CyndiLauper library are both released under the MIT
 
 ## Related Projects
 - [DevsJustWantToHaveFun](https://github.com/vokal/DevsJustWantToHaveFun-macOS): A Mac tool to generate iOS code from the design elements in a `.truecolors` file.
-- [android-res-gen](https://github.com/vokal/android-res-gen): Automatic resource exporter plugin for android projects, including generating styles from a `.truecolors` file.
+- [TrueColors-Android](https://github.com/vokal/TrueColors-Android): Gradle plugin to generate resources for android projects, including  styles, colors and dimensions from a `.truecolors` file.
```

Comment:
You can be against it all you want, but I stick to the Chicago Manual of Style, which says to use the serial comma. 😀